internal/command/describe_type.go

```diff
@@ -126,7 +127,64 @@ func writeKamelet(dw printers.PrefixWriter, kamelet *v1alpha1.Kamelet, printDeta
 		dw.WriteAttribute("Description", kamelet.Spec.Definition.Description)
 	}
 
+	dw.WriteAttribute("Provider", extractKameletProvider(kamelet))
 	dw.WriteAttribute("Phase", string(kamelet.Status.Phase))
+
+	dw.WriteLine()
+	writeKameletProperties(dw, kamelet, printDetails)
+}
+
+func writeKameletProperties(dw printers.PrefixWriter, kamelet *v1alpha1.Kamelet, printDetails bool) {
+	label := "Properties"
+	if printDetails {
+		section := dw.WriteAttribute("Properties", "")
+		maxLen := getMaxPropertyNameLen(kamelet.Spec.Definition.Properties)
+		format := "%-" + maxLen + "s %-8s %-8s %s\n"
+		section.Writef(format, "Name", "Required", "Type", "Description")
```

Comment:
We could safe some screen real estate by shortening the "Required" column to "Req" and then put a ✓ for 'true' and a space for 'false'.

Reply:
✓